Fixed typos in documentation (#6871)
qmk/qmk_firmware: docs/arm_debugging.md

## Installing the software

The main objective here is to get the MCU Eclipse IDE correctly installed on our machine. The necessary instructions are derived from [this](https://gnu-mcu-eclipse.github.io/install/) install guide.

### The xPack Manager

This tool is a software package manager and it is used to help us get the necessary dependencies.

> XPM runs using Node.js so grab that form [here](https://nodejs.org/en/). After installation, open a terminal and type `npm -v`. A reply with the version number means that the installation was successful.

Reviewer: ```suggestion
XPM runs using Node.js so grab that from [here](https://nodejs.org/en/). After installation, open a terminal and type `npm -v`. A reply with the version number means that the installation was successful.
```

`xpm install --global @gnu-mcu-eclipse/windows-build-tools`

### Programmer/Debugger Drivers

> Now its the time to install your programmer's drivers. This tutorial was made using an ST-Link v2 which you can get from almost anywhere.

Reviewer: ```suggestion
Now it's time to install your programmer's drivers. This tutorial was made using an ST-Link v2 which you can get from almost anywhere.
```
